daisy_workflows/export/disk_export.wf.json

```diff
@@ -44,7 +44,8 @@
           "Name": "disk-${NAME}",
           "SourceImage": "${export_instance_disk_image}",
           "SizeGb": "${export_instance_disk_size}",
-          "Type": "${export_instance_disk_type}"
+          "Type": "${export_instance_disk_type}",
+          "FallbackToPdStandard": true
```

Comment:
Do we want to hardcode this behavior to all workflows or do it programmatically from import/export wrappers? There might be customers using these workflows outside our import/export (gcloud/UI) that will see this change. Might not be a big deal, but a thing to consider.

Reply:
hardcode this behavior to import/export related workflows, otherwise import/export can still fail on some step.